src/Rateable/Rating.php

```diff
@@ -27,6 +27,6 @@ public function rateable()
      */
     public function user()
     {
-        return $this->belongsTo(Config::get('auth.model'));
+        return $this->belongsTo(Config::get('auth.providers.users.model'));
```

Comment:
This change will break on older versions of laravel.  Off the top of my head, maybe we should instead do something like this?

```
public function user()
{
  if (!$userClass = Config::get('auth.model')) {
    $userClass = Config::get('auth.providers.users.model'));
  }

  return $this->belongsTo($userClass);
}
```